sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslNestedRowsTest.java

```diff
@@ -35,6 +36,8 @@
   @Rule public ExpectedException exceptions = ExpectedException.none();
 
   @Test
+  @Ignore(
+      "[BEAM-9378] This is a test of the incorrect behavior that should not work but does because calcite flattens the row.")
```

Comment:
Nit: I am slightly not favor of leaving comments for why this is ignored but the comments itself is a very briefly description. I feel like moving the explanations to JIRA and make JIRA more informatively is more important.

Reply:
I wonder - is there value in continuing to run these tests even though the behavior is unexpected?

Reply:
That is, presumably something is relying on the behavior if we wrote a test for it.

Reply:
Turned the tests that are verifying incorrect behavior back on, but left the comment. The remaining "@Ignore" comments basically restate the title of the jira item